OPEN_PROBLEMS/DECENTRALIZED_DATA_DELIVERY_MARKETS.md

```diff
@@ -0,0 +1,511 @@
+- [Proof-of-Prestige: A Useful Work Reward System for Unverifiable Tasks](INSERT URL)
+- [Mechanisms for Outsourcing Computation via a Decentralized Market](INSERT URL)
+- Other Gradual Fair Exchange Class of constructions
+
+**Known shortcomings of these approaches:**
+*   Not a direct shortcoming, but a challenge is that each participant my have different levels of sensitivity with regards to how much they are willing to rely on trust vs. strict verifiability. 
+
+#### Privacy focused
+
+There are multiple solutions with the goal of delivering readers privacy and writers privacy. These incur an additional cost in setup and/or latency in favor of the additional privacy preserving benefits. Disclaimer: At the time of writing, we haven't dived as deep into these set of solutions.
+
+- [Blockchain based Privacy-Preserving Software Updates with Proof-of-Delivery for Internet of Things](INSERT URL)
+- [SilentDelivery: Practical Timed-delivery of Private Information using Smart Contracts](INSERT URL)
+- [Bulletproofs+: Shorter Proofs for Privacy-Enhanced Distributed Ledger](INSERT URL)
+
+#### Traditional approaches (close to Web 2.0 world)
+
+
+These approaches suggest novel ways to add integrity checks to data transferred or to promote new ways for content distribution, but they don’t depart from the Web 2.0 world of assumptions such as trusted third parties and central points of control.
+
+- [The multimedia blockchain: a distributed and tamper-proof media transaction framework](INSERT URL)
+- [Reliable Client Accounting for P2P-Infrastructure Hybrids](INSERT URL)
+
+**Known shortcomings of these approaches:**
+*   Not fully aware of the challenges of building a decentralized network
+
+### Known attacks to be mitigated
+
+Here we list the attacks to consider when designing a solution. These are:
+
+*   Malicious actor forces provider to spend bandwidth without issuing payment (also known as Grieving Attack)
+    *   Type: client fraud 
+    *   Consequence: bandwidth-spent 
+*   Malicious actor forces provider to pull the file from storage point (e.g. Filecoin, Cloud Storage, etc), paying the cost without the provider ever getting paid 
+    *   Type: client fraud: 
+    *   Consequence: currency is spent
+*   Provider claims it has sent the file, but actually never did 
+    *   Type: Provider fraud
+    *   Consequence: Client gets penalized without receiving the service
+*   Metering Inflation 
+    *   Providers report to have used more resources to serve content to clients than what they have actually done.
+*   Sybil / Throttle attacks
+    *   Description: A set of nodes collude against a content-provider to try and make him go bankrupt (loss of all its “SLA stake”). They all try to download content from that content provider at the same time, so that Providers start consuming the stake from the content providers. If clients are not paying for the service we take the risk of enabling this attack. 
```

Comment:
```suggestion
    *   Description: A set of nodes collude against a content publisher to try and make it go bankrupt (i.e. trigger a loss of all its “SLA stake”). The colluding nodes simultaneously try to download content offered by a content publisher in a way that causes providers to start consuming the publisher's stake. This attack is possible if clients are not required to pay for service. 
```
I think this is what you meant (i.e. "content provider" should be "content publisher").